components/gallery/GalleryItem.vue

```diff
@@ -49,7 +49,7 @@
             class="gallery-item-media relative"
             :src="getMediaSrc(image)"
             :animation-src="nftAnimation"
-            :mime-type="nftAnimationMimeType || nftMimeType"
+            :mime-type="(nftAnimation && nftAnimationMimeType) || nftMimeType"
```

Comment:
```suggestion
            :mime-type="(nftAnimation && nftAnimationMimeType) || nftMimeType"
```


```suggestion
            :mime-type="(nftAnimation ? nftAnimationMimeType : undefined) || nftMimeType"
```
In case of that `nftAnimation` is not empty but `nftAnimationMimeType` is undefined.